.pre-commit-config.yaml

```diff
@@ -48,5 +48,7 @@ repos:
     -   id: pylint
         name: pylint
         entry: pylint
+        args:
+        - --good-names=urlpatterns
```

Comment:
This is a good case when to change pylint's rules: when there's something that the framework you use enforces, you should respect those principles. Well done!

Nowadays I lean towards creating a proper config file for pylint rather than using args but it's fine for now, it can be done separately.

Another thing to try is https://pypi.org/project/pylint-django/ — I believe that using it will silence a number of default pytest rules that conflict with the django way of doing things.